src/Summoner/Config.hs

```diff
@@ -148,8 +165,8 @@ configT = Config
     ghcVerArr :: Key -> BiToml [GhcVer]
     ghcVerArr = Toml.arrayOf _GhcVer
```

Comment:
Could you also change `lastT` parser couple lines about to use `wrapper` combinator from `tomland`?

* http://hackage.haskell.org/package/tomland-0.3.1/docs/Toml-Bi-Combinators.html#v:wrapper